### PR TITLE
Doc

### PR DESCRIPTION
**Pull request by jens-br**
This PR adds information about the NEST Server kernel settings to the docs (and introduces also small optimizations to the structure of the Readme.md).

### CODE REVIEW

File: docs/user/usage.rst [hunk 1]
```diff
@@ -144,15 +144,37 @@ Simulate neuronal networks
...
+It is possible to choose a randomized seed.
+
+The simulation time can be set as well (in Milliseconds).
+
+
 
 **Code editor**
 
+.. image:: ../_static/img/screenshots/code-editor.png
+  :width: 360px
```
Comment (babsey) [inline docs/user/usage.rst:175]: Maybe more width (400px)

**Review by babsey — COMMENTED** [review]
LGTM. 

Small minor suggestions can be found in code lines.

File: docs/index.rst [hunk 2]
```diff
@@ -85,3 +85,16 @@ Select the appropriate section that fits your needs.
...
+|
+
+**Version info**
+
+On Read the Docs, it is possible to select versions of this documentation.
+This can be done in the left side bar at the very bottom.
+These versions basically relate to the program versions (as found in the GitHub repository).
+This can be noticed when clicking on a the "Edit on Github" text at the top in the middle.
+
+Since the changes between patch level versions (e.g. 2.4.0 and 2.4.1) are usually not noticeable, we show a single branch for the recent minor version releases (e.g. '2.4' for all 2.4.x releases).
```
[the remark this answers is not in the document]
Comment (jens-br) [reply docs/index.rst]: It was only meant as an example with a - more or less - random version number. However, I changed it.

File: docs/user/usage.rst [hunk 3]
```diff
@@ -144,15 +144,37 @@ Simulate neuronal networks
 You can click on the :guilabel:`Simulate` button to start the simulation of your network.
 In the code editor you can have an insight into the generated script code (see below for further information).
 
-|
 
-.. image:: ../_static/img/screenshots/code-editor.png
+
+**Kernel settings**
+
+.. image:: ../_static/img/screenshots/kernel-settings.png
```
[the remark this answers is not in the document]
Comment (jens-br) [reply docs/user/usage.rst:151]: Done.

File: docs/user/usage.rst (hunk 1 above)
Comment (jens-br) [reply docs/user/usage.rst:175]: The width of the image is already at 640px. I cropped the white space on the right (now 547px), maybe this helps a bit. If you think this is still an issue, please send me a message.

**Review by babsey — COMMENTED** [review]
LGTM. 

A personal opinion: The image stretched too long and could be shorter.

Can you remove the section after simulate(), from the image?
I mean keep lines until line number 36.

**Comment by jens-br** [comment]
Removed the lines after `simulate(...)` as suggested.

**Pull request merged**